pallets/services-payment/src/lib.rs

```diff
@@ -191,6 +207,26 @@ pub mod pallet {
 
             Ok(().into())
         }
+
+        /// Call index to set the refund address for non-spent tokens
+        #[pallet::call_index(3)]
+        #[pallet::weight(T::WeightInfo::set_refund_address())]
+        pub fn set_refund_address(
+            origin: OriginFor<T>,
+            para_id: ParaId,
+            refund_address: T::AccountId,
```

Comment:
No way to remove the refund address?

Reply:
good catch

Reply:
I will add that

Reply:
So I added the refund_address as an optional parameter, if None is provided we remove it